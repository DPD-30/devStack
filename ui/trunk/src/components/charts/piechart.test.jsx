import React from 'react'
import { renderToString } from 'react-dom/server'
import { mount } from 'enzyme'
import PagesPieChart from './piechart.jsx'

jest.unmock('./piechart.jsx')


describe('', () => {
    it('PagesPieChart should match snapshot', () => {
        const comp = renderToString(<PagesPieChart isFail={false} isPass={false}/>)

        expect(comp).toMatchSnapshot()
    })

    it('match snapshot with populate state varaiables', () => {
        let component = mount(<PagesPieChart/>)
 
        expect(component).toMatchSnapshot()

        component = mount(<PagesPieChart 
            pieData={{}}
            datasets={{}}
        />)
 
        expect(component).toMatchSnapshot()

        component = mount(<PagesPieChart 
            pieData={{id:1}}
            datasets={{}}
        />)
 
        expect(component).toMatchSnapshot()
    })

})
