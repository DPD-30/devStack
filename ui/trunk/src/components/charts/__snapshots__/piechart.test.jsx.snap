// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagesPieChart should match snapshot 1`] = `
<div class="cardStyle card"
     data-reactroot
>
  <div class="titleContainer">
    <h4 class="titleStyle">
    </h4>
    <p class="subTitleStyle">
    </p>
  </div>
</div>
`;

exports[`match snapshot with populate state varaiables 1`] = `ReactWrapper {}`;

exports[`match snapshot with populate state varaiables 2`] = `ReactWrapper {}`;

exports[`match snapshot with populate state varaiables 3`] = `ReactWrapper {}`;
